bin/moodle-docker-compose.cmd

```diff
@@ -13,6 +13,7 @@ POPD
 SET COMPOSE_CONVERT_WINDOWS_PATHS=true
 
 SET DOCKERCOMPOSE=docker-compose -f "%BASEDIR%\base.yml"
+SET DOCKERCOMPOSE=%DOCKERCOMPOSE% -f "%BASEDIR%\service.mail.yml"
```

Comment:
Ignore my comment :blush:: you're properly use multiple configuration files via `-f`.

Apologies for the noise!
Matteo